src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+    height: 60px;
+    display: flex;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    font-weight: 500;
+    text-transform: uppercase;
+    font-size: 12px;
+    line-height: 14px;
+    color: $main-color;
```

Comment:
The variable `$main-color` is used here, but it's not defined in this file. Ensure that it is defined in a global variables file or imported correctly.